### PR TITLE
lps-102788

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
@gamerson looks like there are some plugin diff issues. I'll clean those up. In the meantime, do you think you can review the other stuff?

- compare bnd test
- update to bnd 4.3.0
- splitting out taglibs. Rich said to target all the stuff that was previously in liferay-ui, so that's what I based it off of.

File: modules/build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import com.liferay.gradle.util.FileUtil
+		import com.liferay.gradle.util.FileUtil
```
Comment (gamerson) [inline modules/build.gradle]: whitespace

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle [hunk 2]
```diff
@@ -28,13 +30,17 @@ dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "2.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "2.0.0"
 #elseif (${liferayVersion.startsWith("7.1")})
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "3.2.2"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.util.freemarker.contributor", version: "2.0.0"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle:34]: what is this dependency used for?

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle [hunk 3]
```diff
@@ -28,13 +30,17 @@ dependencies {
...
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "3.0.2"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "3.1.2"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "3.4.2"
 #elseif (${liferayVersion.startsWith("7.2")})
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib", version: "4.0.15"
+	compileOnly group: "com.liferay", name: "com.liferay.frontend.taglib.util.freemarker.contributor", version: "3.0.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "4.4.0"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "6.0.2"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "4.0.8"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "4.1.0"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle:43]: Does this version match 7.2 GA1 or upcoming release?

Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle:43]: I matched it to 7.2 GA1 BOM. I thought it was originally matched to the GA1 BOM already, but maybe i'm wrong. Or they reuploaded a newer BOM

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle (hunk 2 above)
Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle:34]: I'm not 100% sure. I added these taglibs based on keywords. Freemarker and util haha

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
https://github.com/gamerson/liferay-portal/pull/355

**Pull request closed without merging**